x/permission/types/types.go

```diff
@@ -167,26 +183,25 @@ func (s *Statement) ValidateBasic(resType resource.ResourceType) error {
 	case resource.RESOURCE_TYPE_UNSPECIFIED:
 		return ErrInvalidStatement.Wrap("Please specify the ResourceType explicitly. Not allowed set RESOURCE_TYPE_UNSPECIFIED")
 	case resource.RESOURCE_TYPE_BUCKET:
-		containsCreateObject := false
-		for _, a := range s.Actions {
-			if !BucketAllowedActions[a] {
-				return ErrInvalidStatement.Wrapf("%s not allowed to be used on bucket.", a.String())
-			}
-			if a == ACTION_CREATE_OBJECT {
-				containsCreateObject = true
-			}
-		}
+		//containsCreateObject := false
```

Comment:
You'd better to remove these useless code, rather than comment.

Reply:
Sure.